src/traces/scatter/attributes.js

```diff
@@ -292,14 +292,27 @@ module.exports = {
             ].join(' ')
         },
         dash: extendFlat({}, dash, {editType: 'style'}),
+        backoff: { // TODO: do we want to have a similar option for the start of the line? If so what the name should be?
```

Comment:
I don't have a good name in mind for this, but let's not worry about it for now. If we add this and the name isn't symmetric with `backoff`, we can just reference that name in the description here so people can find it.

Reply:
Addressed in https://github.com/plotly/plotly.js/pull/6297/commits/83b1455d2808e6c5fa0155a7df46d3b8168190a5.